src/components/backup/new.vue

```diff
@@ -352,6 +352,10 @@ export default {
     "title": "Configurer des sauvegardes",
     "steps": [
       {
+        "warning": {
+          "full": "Cette sauvegarde n'inclue pas présentement les liens de formulaires Web. Si vous partagés des liens d'accès public ou partagés des liens directs à vos formulaires Web, nous recommandons fortement que vous fassiez également une sauvegarde complete du système jusqu'à ce que ceci soit adressé. Si vous devez restaurez vos données d'une sauvegarde et vous retrouvez avec des liens d'aperçu qui ne fonctionnent pas, veuillez écrire un message sur {forum} pour recevoir de l'aide.",
```

Comment:
Oh my goodness who translated this, a potato? I'm so embarrassed. 🤦🏻‍♀️ Fixed in Transifex but I guess it can just live on as a reminder to others to participate in the translation process.